client/src/app/managing-body-detail/managing-body-detail.component.ts

```diff
@@ -1,7 +1,6 @@
 import { Component } from '@angular/core';
 
 @Component({
-  selector: 'op-managing-body-detail',
```

Comment:
Yeah revert it.
I did not explain myself well, I meant do delete the empty selectors in each component, like in `app` component, we don't use the `pipes` selector (the array is empty -> we don't use it, so delete it.)